### PR TITLE
Feat/port new langs from upstream

### PR DESCRIPTION
**Pull request by JarbasAl**
ports basque and russian lang resources from https://github.com/MycroftAI/mycroft-core/pull/2961 and https://github.com/MycroftAI/mycroft-core/pull/3014

### CODE REVIEW

File: mycroft/res/text/eu-eu/message_synching.clock.dialog [hunk 1]
```diff
@@ -0,0 +1 @@
+< < < SINKRONIZATZEN < < <
```
Comment (NeonDaniel) [inline mycroft/res/text/eu-eu/message_synching.clock.dialog:1]: Should this be cleaned to alphas?

**Review by NeonDaniel — APPROVED** [review]
Can't comment on accuracy, only comment is if the dialogs containing `<` should be cleaned to alpha

File: mycroft/res/text/eu-eu/message_synching.clock.dialog (hunk 1 above)
Comment (JarbasAl) [reply mycroft/res/text/eu-eu/message_synching.clock.dialog:1]: no, this is what is sent to the mk1 faceplate, the < are part o the scrolling message and added on purpose

**Comment by ChanceNCounter** [comment]
I'm counting on the upstream review and green tests on this one. I'm confident, having watched the upstream PR go through.

**Pull request merged**